### ISSUE TITLE
Scheduled Builds Failing

### ISSUE THREAD

**Issue by danielgroves** [gitlabhq/gitlab-ci#519]
I've not managed to find this issue elsewhere. 

Scheduled builds are failing on a new Gitlab CI install, running 5.3.0. The following is being appended to my /var/mail/gitlab_ci file. 

```
From gitlab_ci@XXX.danielgroves.net  Sat Dec 27 18:05:05 2014
Return-Path: <gitlab_ci@XXX.danielgroves.net>
X-Original-To: gitlab_ci
Delivered-To: gitlab_ci@XXX.danielgroves.net
Received: by XXX.danielgroves.net (Postfix, from userid 1001)
	id 848C51654B4; Sat, 27 Dec 2014 13:05:05 -0500 (EST)
From: root@XXX.danielgroves.net (Cron Daemon)
To: gitlab_ci@XXX.danielgroves.net
Subject: Cron <gitlab_ci@XXX> /bin/bash -l -c 'cd /home/gitlab_ci/gitlab-ci && RAILS_ENV=production bundle exec rake schedule_builds --silent'
Content-Type: text/plain; charset=ANSI_X3.4-1968
X-Cron-Env: <SHELL=/bin/sh>
X-Cron-Env: <HOME=/home/gitlab_ci>
X-Cron-Env: <PATH=/usr/bin:/bin>
X-Cron-Env: <LOGNAME=gitlab_ci>
Message-Id: <20141227180505.848C51654B4@XXX.danielgroves.net>
Date: Sat, 27 Dec 2014 13:05:05 -0500 (EST)

rake aborted!
NoMethodError: undefined method `last_build' for #<Project:0x007f51c8c171b0>
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/activemodel-4.0.10/lib/active_model/attribute_methods.rb:439:in `method_missing'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/activerecord-4.0.10/lib/active_record/attribute_methods.rb:168:in `method_missing'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:5:in `block in perform'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:4:in `each'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:4:in `perform'
/home/gitlab_ci/gitlab-ci/lib/tasks/schedule_builds.rake:3:in `block in <top (required)>'
Tasks: TOP => schedule_builds
(See full trace by running task with --trace)
```

Other similar issues seem to be around the `bundle` command not being found… that's not the issue here as it's clearly getting further than that. 

Also run the command manually with the following output:

```
gitlab_ci@XXX:/root$ /bin/bash -l -c 'cd /home/gitlab_ci/gitlab-ci && RAILS_ENV=production bundle exec rake schedule_builds --trace'
** Invoke schedule_builds (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute schedule_builds
rake aborted!
NoMethodError: undefined method `last_build' for #<Project:0x007f4d65e52bd0>
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/activemodel-4.0.10/lib/active_model/attribute_methods.rb:439:in `method_missing'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/activerecord-4.0.10/lib/active_record/attribute_methods.rb:168:in `method_missing'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:5:in `block in perform'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:4:in `each'
/home/gitlab_ci/gitlab-ci/lib/scheduler.rb:4:in `perform'
/home/gitlab_ci/gitlab-ci/lib/tasks/schedule_builds.rake:3:in `block in <top (required)>'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `call'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `block in execute'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `each'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `execute'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'
/usr/local/lib/ruby/2.1.0/monitor.rb:211:in `mon_synchronize'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:165:in `invoke'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:150:in `invoke_task'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `each'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block in top_level'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:115:in `run_with_threads'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:100:in `top_level'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:78:in `block in run'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:75:in `run'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/bin/rake:33:in `<top (required)>'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/bin/rake:23:in `load'
/home/gitlab_ci/gitlab-ci/vendor/bundle/ruby/2.1.0/bin/rake:23:in `<main>'
Tasks: TOP => schedule_builds
```

**Comment by jacobvosmaer** [comment]
Yes this is broken, I have noticed it too. Seems to be since 5.3.0.

**Comment by jacobvosmaer** [comment]
gitlab.com issue https://gitlab.com/gitlab-org/gitlab-ci/issues/84

**Comment by jacobvosmaer** [comment]
Should be fixed on master by 9d898f8b0508f43df.

**Comment by danielgroves** [comment]
Ah, excellent. Thanks @jacobvosmaer, I hadn't come across those. I guess this will be patched in the next release then?

**Comment by vsizov** [comment]
fixed in master by 9d898f8b0508f43dfffb694d5f480a09bf400a0f
